distribution/zip/pom.xml

```diff
@@ -13,7 +13,6 @@
     <name>Elasticsearch ZIP Distribution</name>
 
     <properties>
```

Comment:
I was keeping those for all the distributions, we can remove those properties sections if we want. ill nuke em.